backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/interfaces/I_C_OrderLine.java

```diff
@@ -43,4 +43,8 @@
 	void setHandOver_Location_ID(int handoverLocationId);
 	int getHandOver_Location_ID();
 	// @formatter:on
+
+	String COLUMNNAME_M_HU_PI_Item_Product_ID = "M_HU_PI_Item_Product_ID";
```

Comment:
we already have this, in `de.metas.handlingunits.model.I_C_Order`

Reply:
Removed.